src/database/migrations/20190903175944-create-user.js

```diff
@@ -6,12 +6,12 @@ module.exports = {
       type: Sequelize.INTEGER,
       allowNull: false
     },
-    firstname: { type: Sequelize.STRING, allowNull: false },
-    lastname: { type: Sequelize.STRING, allowNull: false },
-    email: { type: Sequelize.STRING, allowNull: false },
+    firstName: { type: Sequelize.STRING, allowNull: false },
+    lastName: { type: Sequelize.STRING, allowNull: false },
+    email: { type: Sequelize.STRING, allowNull: false, unique: true },
     password: { type: Sequelize.STRING, allowNull: false },
     createdAt: { type: Sequelize.DATE, allowNull: false },
     updatedAt: { type: Sequelize.DATE, allowNull: false }
   }),
-  down: (queryInterface) => queryInterface.dropTable('Users')
+  down: queryInterface => queryInterface.dropTable('Users')
 };
```

Comment:
Why did you change this from `create-users` to `create-user`?

Reply:
I created another table as suggested. But I used `user` instead of `users` since Sequelize will automatically pluralize it.

Reply:
Did you delete the current migration file before creating the new one? @hustlaviola

Reply:
Yes